core/modules/replica/AddReplicaQservMgtRequest.h

```diff
@@ -131,6 +122,32 @@ class AddReplicaQservMgtRequest
       */
     void notify(util::Lock const& lock) final;
 
+private:
+
+    /**
+     * Construct the request with the pointer to the services provider.
+     *
+     * @param serviceProvider
+     *   reference to a provider of services
+     *
+     * @param worker
+     *   the name of a worker
+     *
+     * @param chunk
+     *   the chunk number
+     *
+     * @param databases
+     *   the names of databases
+     *
+     * @param onFinish
+     *   callback function to be called upon request completion
+     */
```

Comment:
Is this recommended somewhere? I preferred the previous comment block to this one as this one uses way more space than it needs. Pretty much all of the parameters are self explanatory, " serviceProvider reference to a provider of services" is pointless as documentation since the documentation says exactly the same thing as the code. Only the onFinish comment is somewhat useful. The first version only ate up 9 lines of header file instead of 18, which bothers me in that it takes that much longer to scan through the header files to find the important things.

Reply:
:100:  This kind of comment is USELESS DISTRACTING GARBAGE and FILLS ME WITH RAGE.  Write your code for PEOPLE to read.

Reply:
I will also point out the raging internal consistency between being bothered by license headers but not by this kind of faffery.  How can you sleep at night?

Reply:
I don't care much about "space utilization", it's free. Putting parameter descriptions (which is permitted by Oxygen) onto a separate line fixed a need to align descriptions with was causing me troubles in the previous version of the comments.
Regarding self-explanatory, you may be right here. However, I need to document all parameters, not just one.

Reply:
Many linefeeds gave their lives...

Reply:
To @fritzm , I always have a habit to extensively comment my code, even if that might seem an overkill to others. I do this for many years ... and for myself. Later, when I'm looking at a code I usually first look at its structure, then I  always read comments, and then (if needed) read what that code does. This helps me to remember details after I forget it (which is a case for all of us).
As far as "faffery" is concerned, adding comments to the code is really cheep compared to the time & efforts spent to understand which code to write how to write it.

Reply:
I have no problem with comments.  I have issue with "content-free" comments, and formatting that makes it harder for me to have more code on the screen at once.

Reply:
I want to read the _code_.  I want comments when the code is tricky or non-obvious (hopefully not that common!)

Reply:
Regular expressions inside an IDE do the job. For example (in Netbeans), this pattern:
```
@param (.+)\s-\s(.+)$
```
And the replacement:
```
@param $1\n     *   $2
```

Reply:
I glad to hear this won't be hard for you to fix! :-)

Reply:
(Don't take me too seriously.  This objection is not "blocking" for review.  I just respect you less.)

Reply:
I'm glad that I still have some credit of respect 😄 
To be serious, when it comes to the code I tend to be consistent in a way I write it which means: 1) always explaining each class/struct, 2) commenting non-trivial methods, 3) when commenting a method then I comment all parameters o the method, 4) commenting non-trivial members of a class/struct, 5) commenting statements or groups of statements in method bodies when it's not clear what they do (I just do not make an assumption that people should waste there tim trying to figure what that code is meant to do).
I do value the "self-explanatory" approach to naming identifiers. However this has its own limits.

Reply:
In the end, tis effort was to make everything look consistent (I had a few different styles in the previous version of the code, which was bothering me).

Reply:
There is something to be said for consistency...  I've taken more than enough time/space now to soapbox my opinion.  Peace, friend!

Reply:
@jgates108 

Regarding 1) earlier (beginning of this year, or end of the last year) I was already criticized by one of reviewers that I didn't document all parameters of a method (I missed one or two). So, now I'm hearing a conflicting requirement. I would rather stick to the current model of documenting all of them.

Regarding 2), your are making a good point on a possibility of changing a type of chunk identifiers. Coincidently I already have this in my TODO list. Leaving `@param chunk` w/o any description is fine with me. Though, I would rather revisit each method (at lest factory methods or constructors) where the chunk identifier are passed and expand parameter descriptions with an intended purpose of passing those numbers. For example: "an identifier of a chunk to be deleted from a worker" where it's not clear from a context why the chunk number is needed here.

Regarding 3) the Replication System was actually designed as an extendable framework which can be modified by anyone of us (or not just by us - by the people who'll maintain this software in many years from now). Having some in-code documentation (I hope) will help. It also helps to myself in the future should I need to modify this code. My memory is fading quicker than I want it to 😄 

Anyway, thank you very much for the good comments and suggestions!